web/themes/custom/recipe_theme/src/sass/view/_user_account_page.scss

```diff
@@ -0,0 +1,59 @@
+.user__name,
+.user__field-first-name {
+  .field__label,
+  .field__item {
+    display: flex;
+    justify-content: center;
+    font-family: 'Dosis', sans-serif;
+    font-weight: 300;
```

Comment:
If you overwrite this in `field__label` why not just apply this rule to `field__item`?